CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ virtualenv -p python3 venv && \
 If you're planning to contribute code, it's a good idea to enable the standard Git hooks so that build scripts run before you commit. That way, you can see if basic tests pass in a few seconds rather than waiting a few minutes to watch them run in CircleCI.
 
 ```bash
-./dev-scripts/hooks/enable_hooks
+./dev-scripts/hooks/enable-hooks
```

Comment:
I took the opportunity to fix the typo ;)

Reply:
Thanks! I try to keep the PRs [narrowly scoped](https://mtlynch.io/code-review-love/#5-narrowly-scope-changes), so I moved this change to its own PR:

https://mtlynch.io/code-review-love/#5-narrowly-scope-changes